### DISCUSSION TITLE
Getting Started with brew on Mac

### DISCUSSION THREAD

**Discussion by asadfaizi-github** [Homebrew/discussions#5038] [Getting started]
### Output of `brew config`

```shell
curl: (22) The Requested URL returned error: 403
```

### Output of `brew doctor`

```shell
curl: (22) The Requested URL returned error: 403
```

### Description of issue

I am trying to install brew on Macbook Pro (13.6.2) with Apple M1 Pro chip

When I execute the command:

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

I get:  curl: (22) The Requested URL returned error: 403

**Comment by p-linnane** [comment]
This is a network problem, not an issue with Homebrew. You're receiving a 403 Forbidden error from GitHub's servers.
